modules/apps/jaggery/emm/src/emm/units/policy-view-edit/public/js/policy-create.js

```diff
@@ -22,6 +22,7 @@ var stepForwardFrom = {};
 var stepBackFrom = {};
 var policy = {};
 var configuredOperations = [];
+var currentlyEffected = {};
```

Comment:
Since this is a associated array the usage will be as follows,
currentlyEffected.users / currentlyEffected.roles 
Which holds the list of users or list of roles that is effected by the current policy.
Any suggestions for improvements ?